cogs/logging.py

```diff
@@ -401,23 +495,69 @@ def __init__(self, bot):
                 member, after.channel, 'joined'
             )
             for channel in vc_channels:
-                channel = self.bot.get_channel(channel)
-                await channel.send(embed=local_embed)
+                try:
+                    channel = self.bot.get_channel(channel)
+                    await channel.send(embed=local_embed)
+                except Exception as e:
+                    self.bot.logger.info(
+                        f'Error logging voice join in'
+                        f' channel {channel}, error: {e}'
+                    )
         elif after.channel is None and before.channel:
             local_embed = embeds.VoiceChannelStateEmbed(
                 member, before.channel, 'left'
             )
             for channel in vc_channels:
-                channel = self.bot.get_channel(channel)
-                await channel.send(embed=local_embed)
+                try:
+                    channel = self.bot.get_channel(channel)
+                    await channel.send(embed=local_embed)
+                except Exception as e:
+                    self.bot.logger.info(
+                        f'Error logging voice leave in'
+                        f' channel {channel}, error: {e}'
+                    )
         elif before.channel != after.channel:
             local_embed = embeds.VoiceChannelMoveEmbed(
                 member, before.channel, after.channel
             )
             for channel in vc_channels:
-                channel = self.bot.get_channel(channel)
-                await channel.send(embed=local_embed)
-
-
+                try:
+                    channel = self.bot.get_channel(channel)
+                    await channel.send(embed=local_embed)
+                except Exception as e:
+                    self.bot.logger.info(
+                        f'Error logging voice move in'
+                        f' channel {channel}, error: {e}'
+                    )
+                    
+    async def on_guild_channel_delete(channel):
+        """
+        attempts to remove deleted channel from the logging databases
+        """
+        server_del = channel.guild.id
+        try:
```

Comment:
This try: catch block is missing catch statements, I would personally say to indent the if block by one and add a catch after.

Reply:
Do you need a catch after? if it fails I just assume it's not in the database list

Reply:
Yes because without a catch, python will throw an error and not let you run it

You can even just do a catch like
```
except Exception:
    self.bot.logger.warning(f'Issue removing channel')
```

Reply:
How about
`except Exception:
    continue`
because it would be a pain to see every channel that gets deleted

Reply:
It would only show every channel that isn't deleted due to an exception correct?

Reply:
Wouldn't every channel that's not in the database throw an error?

Reply:
No it wouldn't, it would return false to `success` because the DB should return false.

Also after looking into it the DB is already logging the actual exception so something like this would just add more context to that error logging
```
Exception:
    self.bot.logger.warning(f'Issue removing channel after deletion')
```

Reply:
Because not deleting any rows isn't an error that gets thrown